src/app/app.component.ts

```diff
@@ -889,6 +889,15 @@ export class AppComponent implements OnInit {
 
   public toggleExpandCollection = (collection: Collection) => {
     this.expandedCollectionId = this.expandedCollectionId === collection.id ? null : collection.id;
+
+    /*
+     * If we just closed an expanded collection panel, as opposed to
+     * opening a new one, we detect visibility again since that may
+     * reveal hidden collections.
+     */
+    if (this.expandedCollectionId === null) {
+      this.setCollectionsViewImageVisibility();
```

Comment:
this is being called when we closed an expanded collection panel but it does not meet the conditions on line 929 `if (!isCollectionScrolledPastLeft && !isCollectionScrolledPastRight) {` as isCollectionScrolledPastRight is `true` when in this state, so no further collections become visible.

Reply:
How about now?

Reply:
Nope.